website/addons/figshare/tests/test_views.py

```diff
@@ -23,6 +25,15 @@
 figshare_mock = create_mock_figshare(project=436)
 
 
+def assert_urls_equal(url1, url2):
```

Comment:
This is already defined in `tests.test_addon` (though it should probably be moved to `tests.base` or `tests.utils`).